src/main/java/io/percy/selenium/Percy.java

```diff
@@ -164,6 +173,81 @@ public void snapshot(String name, Map<String, Object> options) {
         postSnapshot(domSnapshot, name, driver.getCurrentUrl(), options);
     }
 
+    /**
+     * Take a snapshot and upload it to Percy.
+     *
+     * @param name      The human-readable name of the screenshot. Should be unique.
+     */
+    public void screenshot(String name) throws UnsupportedOperationException {
+        Map<String, Object> options = new HashMap<String, Object>();
+        screenshot(name, options);
+    }
+
+    /**
+     * Take a snapshot and upload it to Percy.
+     *
+     * @param name      The human-readable name of the screenshot. Should be unique.
+     * @param options   Extra options
+     */
+    public void screenshot(String name, Map<String, Object> options) throws UnsupportedOperationException {
+        if (!isPercyEnabled) { return; }
+        List<String> driverArray = Arrays.asList(driver.getClass().toString().split("\\$")); // Added to handle testcase (mocked driver)
```

Comment:
make sure the RemoteDriver is supported by `Environment.java`

Reply:
verify this